### PR TITLE
NodeJs Scripts to load tables

### PR DESCRIPTION
**Pull request by angelinrashmi2000**
load_entities loads the input json file to entities table hosted in testmusicmachine.northeurope.cloudapp.azure.com. This table is exposed as API to Group2.

### CODE REVIEW

File: Scripts_to_load_tables/load_entities.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+////// THIS SCRIPT LOADS THE INPUT JSON FILE INTO ENTITIES TABLE.THIS TABLE IS EXPOSED TO GROUP 2 THORUGH REST API'S //////////////////////////
+var fs = require("fs");
+var pg = require("pg");
+var content = fs.readFileSync("dbpedia_composer.json"); //296
+var jsonContent = JSON.parse(content);
+var cn = {
```
Comment (sacdallago) [inline Scripts_to_load_tables/load_entities.js]: Please move this logic to a global config file which is .gitignored but for which you provide a template https://github.com/MusicConnectionMachine/StructuredData/pull/24/files#r106035398

**Comment by kordianbruck** [comment]
@MusicConnectionMachine/group-1 also has been waiting for 2 days already...

**Comment by sacdallago** [comment]
I think this is old anyway? Should be moved to [the api](https://github.com/MusicConnectionMachine/api)?

**Comment by angelinrashmi2000** [comment]
Yes this is the old load script to populate the entities database. This will be moved to api repo.

**Pull request closed without merging**